tests/test_coordinates.py

```diff
@@ -524,6 +525,26 @@ def test_roundtrip_GEO_GEI(self):
         np.testing.assert_allclose(got.data, expected.data)
         np.testing.assert_equal(got.dtype, 'GEO')
 
+    def test_roundtrip_loop(self):
+        """Roundtrip should yield input as answer (loop over systems)"""
+        tt = Ticktock(['2002-02-02T12:00:00'], 'ISO')
+        irbframes = list(spc.SYSAXES_TYPES.keys())
+        # exclude frames either not supported by IRBEM or that aren't cartesian
+        for key in ['ECIMOD', 'GDZ', 'RLL', 'CDMAG', 'SPH']:
+            irbframes.pop(irbframes.index(key))
+        for insys in irbframes:
+            for outsys in irbframes:
+                expected = spc.Coords([1, 2, 4], insys, 'car',
+                                      use_irbem=True, ticks=tt)
+                stage1 = expected.convert(outsys, 'car')
+                got = stage1.convert(insys, 'car')
+                np.testing.assert_allclose(got.data, expected.data, atol=0.001)
```

Comment:
Here (and on line 546), using err_msg and verbose would give more information about what in/out combination failed--otherwise it's going to be really opaque.

Reply:
Do we have an example for that or do I have to burn brain cycles and think about it?

Reply:
Try this for line 541:

```python
np.testing.assert_allclose(got.data, expected.data, atol=0.001, verbose=True,
    err_msg='insys: {} outsys: {}'.format(insys, outsys))
```

And for line 546:
```python
np.testing.assert_equal(got.dtype, insys, verbose=True,
    err_msg='insys: {} outsys: {}'.format(insys, outsys))
```

Reply:
Ah, gotcha. That looks like a worthwhile update. Will push shortly.